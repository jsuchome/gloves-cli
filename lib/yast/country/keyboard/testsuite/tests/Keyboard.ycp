/**
 * tests for Keyboard.ycp constructor
 * Author: jsuchome@suse.cz
 * $Id: Keyboard.ycp 39978 2007-08-06 10:46:16Z jsuchome $
 */
{

    // testedfiles: Keyboard.ycp

    include "testsuite.ycp";

    map READ = $[
	"sysconfig" : $[
	    "keyboard" : $[
		"YAST_KEYBOARD"	: "czech-qwerty,pc104",
	    ],
	],
	"target" : $[
	    "size"		: 1,
	    "yast2"		: $[
		"czech-qwerty"	: [
		    "Czech (qwerty)",
		    $[
			"pc104": $[
				"ncurses": "cz-lat2-us.map.gz",
				"compose" : "latin2"
			],
		    ]
		],
	    ],
	    "tmpdir"		: "/tmp",
	    "ycp"		: $[
		"XkbLayout"	: "cz_qwerty,us",
	    ],
	],
	"probe"	: $[
	    "architecture"	: "i386"
	],
    ];
    map EXECUTE	= $[
	"target" : $[
	    "bash_output"	: $[],
	],
    ];

    TESTSUITE_INIT([READ,$[],EXECUTE], nil);

    import "Keyboard";

}
