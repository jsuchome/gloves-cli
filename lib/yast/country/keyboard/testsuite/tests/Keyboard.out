Read	.target.yast2 "keyboard_raw.ycp" $["czech-qwerty":["Czech (qwerty)", $["pc104":$["compose":"latin2", "ncurses":"cz-lat2-us.map.gz"]]]]
Read	.target.size "/usr/sbin/xkbctrl" 1
Execute	.target.bash "/usr/sbin/xkbctrl cz-lat2-us.map.gz > /tmp/xkbctrl.out" 0
Read	.target.ycp "/tmp/xkbctrl.out" $["XkbLayout":"cz_qwerty,us"]
